app/test_split_integer.py

```diff
@@ -2,20 +2,24 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(20, 4)) == 20
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    ext = split_integer(30, 5)
```

Comment:
According to the checklist, avoid creating unnecessary variables for simple tests. Instead of assigning the result to `ext`, you can directly use `assert len(split_integer(30, 5)) == 5`.